app/queries/live_edition_blocking_draft_edition.rb

```diff
@@ -12,10 +12,9 @@ def self.call(content_id, base_path, document_type)
 
       conflicts = Edition.with_document
         .where(base_path: base_path, content_store: :live)
-        .where.not(
-          "documents.content_id": content_id,
-          document_type: SubstitutionHelper::SUBSTITUTABLE_DOCUMENT_TYPES,
-        ).pluck(:id)
+        .where.not("documents.content_id": content_id)
```

Comment:
```suggestion
        .where.not(documents: { content_id: content_id })
```

I think this is equivalent, which might be nicer to read.